### ISSUE TITLE
Clarification - relating subgenera

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks_doc#16]
Currently, there is a soft validation warning for both a genus and its child subgenus if the taxon is nominotypical (e.g. Rhizophagus subgenus Rhizophagus and genus Rhizophagus), that shows up even if the parent genus and the child are identical in every way. 

The message is "The original genus does not match with the original genus of coordinated genus" (for subgenus) and "The original genus does not match with the original genus of coordinated subgenus" (for genus). 

Basically, is the "method" for relationshipping subgenera to create a clone of the genus exactly, just with a parent one level down? Or is there something we need to change to make that Soft Validation go away?

Examples:
Genus:
![image](https://user-images.githubusercontent.com/11411328/77962743-cc3be580-72a1-11ea-8b6d-06ed08854602.png)

Subgenus:
![image](https://user-images.githubusercontent.com/11411328/77962698-b75f5200-72a1-11ea-96ee-ffffc90603f5.png)

**Comment by proceps** [comment]
The model have to be updated. This is on my list.

**Comment by proceps** [comment]
I opened a new issue in TW 
https://github.com/SpeciesFileGroup/taxonworks/issues/1399
